### PR TITLE
{bio}[foss/2020b] OrthoFinder v2.5.2, FastME v2.1.6.1, MCL v14.137, ... w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)
requires ~#12079~ (BLAST+)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3547.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/fb043b10066a7165ab47d40d1b483137 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node2691.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/c08c3b1987e8389b1a1443677c9c0981 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3111.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/eec5e4ec9cea077f72489f8767a1d2ac for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 5 out of 5 (5 easyconfigs in total)
node3507.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/ca62d2156326c16b9fe2da335184eabc for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/f/FastME/FastME-2.1.6.1-GCC-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description = "FastME: a comprehensive, accurate and fast distance-based phylogeny inference program."
+
+toolchain = {'name': 'GCC', 'version': '10.2.0'}
+
+source_urls = ['https://gite.lirmm.fr/atgc/FastME/-/archive/v%(version)s/']
+sources = ['FastME-v%(version)s.tar.gz']
+checksums = ['fae0609576873cb137ad4f63529b3ccba8f747b857c23c7870236aba440076ca']
+
+builddependencies = [
+    ('binutils', '2.35'),
```
Comment (smoors) [inline easybuild/easyconfigs/f/FastME/FastME-2.1.6.1-GCC-10.2.0.eb:16]: @boegel binutils is already included in GCC toolchain?

**Comment by verdurin** [comment]
Test report by @verdurin
**SUCCESS**
Build succeeded for 8 out of 8 (5 easyconfigs in total)
nuc.lan - Linux Fedora 32, x86_64, Intel(R) Core(TM) i7-8650U CPU @ 1.90GHz, Python 3.8.7
See https://gist.github.com/91632f756e2590751bcfd1f78da108de for a full test report.

File: easybuild/easyconfigs/f/FastME/FastME-2.1.6.1-GCC-10.2.0.eb (hunk 1 above)
Comment (Micket) [reply easybuild/easyconfigs/f/FastME/FastME-2.1.6.1-GCC-10.2.0.eb:16]:
```suggestion
```
as one easy click

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3580.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/81a351952049cbc78a4f36994a9307f1 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node3154.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz (skylake_avx512), Python 3.6.8
See https://gist.github.com/229efb6565b31b04fe4b9f3d325d66ac for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
node2697.swalot.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 3.6.8
See https://gist.github.com/57052cd02946f46d3df9f50b13f81800 for a full test report.

**Review by Micket — APPROVED** [review]
lgtm

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**